### PR TITLE
Improve EinsteinHelfand class

### LINKED ISSUE #53: Automatically calculate slope with viscosity runner
Currently the viscosity class will calculate the viscosity function but not automatically find the slope of it. With some guardrails, that'd be a nice feature.

### PR DESCRIPTION
**Pull request by orionarcher**
This PR:
- [x] Fixes a bug in the viscosity implementation.
- [x] Adds a plotting function for the EinsteinHelfand class
- [x] Allows automatic calculation of viscosities by specifying a fitting region
- [x] Slightly improves documentation
- [x] Adds data for a real system with known viscosity and tests against it (value is within 25%)

Fixes #53, #41 (partially), #28

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [ ] CHANGELOG updated? NA
 - [x] Issue raised/referenced?

### CODE REVIEW

File: transport_analysis/viscosity.py [hunk 1]
```diff
@@ -203,7 +209,7 @@ def _conclude(self):
             )
 
             # square and sum each x(, y, z) diff per particle
-            sq_diff = np.square(diff).sum(axis=-1)
+            sq_diff = np.square(diff).mean(axis=-1)
```
Comment (orionarcher) [inline transport_analysis/viscosity.py:212]: @hmacdope @xhgchen, this is the most significant change in this PR, it fixes a bug in the initial implementation.

Previously, we were calculating viscosity in the X, Y, and Z directions and then summing the results, I believe the correct behavior is to take the mean of all three. 

Viscosity should be invariant to rotation, thus viscosity in X and Y and Z should all be the same. It does not make sense in my mind to take the sum of all three. Additionally, the values are much closer to the benchmarking data when we take the mean.

Comment (hmacdope) [reply transport_analysis/viscosity.py:212]: Yep thats an implementation error. Thanks for picking this up.

**Review by hmacdope — APPROVED** [review]

**Comment by hmacdope** [comment]
This has changed the expected value of the step traj test, if you can fix that up would be awesome.

File: transport_analysis/viscosity.py (hunk 1 above)
Comment (orionarcher) [reply transport_analysis/viscosity.py:206]: Thanks for the sanity check.

Comment (xhgchen) [reply transport_analysis/viscosity.py:222]: Great catch! Thanks for the fix and improvements @orionarcher! Appreciate the review, @hmacdope!

**Pull request merged**